AAC templates: Removing unwanted metadata (#52)
MicrosoftDocs/content-templates: templates/org/aac/aac-article-title-content.md

<!-- 
Don't add metadata to this Markdown file.
Use the browser header template to create a YAML file that contains your metadata. 

> Add a brief introductory paragraph with no heading. 

Reviewer: ```suggestion
Add a brief introductory paragraph with no heading.
```

Cost Optimization focuses on ways to reduce unnecessary expenses and improve operational efficiencies. For more information, see [Design review checklist for Cost Optimization](/azure/well-architected/cost-optimization/checklist).

> <!-- 

Reviewer: ```suggestion
<!--
```

> Explain Cost Optimization considerations. 
> Link to the pricing calculator (https://azure.microsoft.com/pricing/calculator), with this architecture prepopulated. 
> Include the major cost-driving components, a typical scale or throughput, and recommended SKUs. 

Reviewer: ```suggestion
Explain Cost Optimization considerations.
Link to the pricing calculator (https://azure.microsoft.com/pricing/calculator), with this architecture prepopulated.
Include the major cost-driving components, a typical scale or throughput, and recommended SKUs.
```

<!-- Explain key performance considerations, beyond the typical. -->

## Deploy this scenario

> <!-- 
> This section is required. 
> Link to a template or script that explains how to deploy the solution. 
> Include source code, deployment code, and instructions for how to test the scenario. 
> -->

Reviewer: ```suggestion
<!--
This section is required.
Link to a GitHub-hosted template or script that explains how to deploy the solution.
The repo will contain all of the source code, deployment code, and instructions for how to try the scenario in the customer's own subscription.
-->
```
